samples/graph-activity-feed/csharp/ActivityFeedNotification/Views/Home/GroupChatNotification.cshtml

```diff
@@ -28,7 +27,7 @@
                 $(targetBox).show();
             });
         });
-        microsoftTeams.initialize();
+        microsoftTeams.app.initialize();
         var token = localStorage.getItem("accessToken");
         var username = "";
         var userId = "";
```

Comment:
again below you are using `microsoftTeams.getContext` instead of `microsoftTeams.app.getContext`. Can you verify that you have migrated all the relevant SDK calls to the v2 implementation.

Will you let me know when you've completed that, as I'll postpone reviewing the rest of this pull request until then